README.md

```diff
@@ -28,6 +33,17 @@ You can also pass the name of your project to the tool using the `--name` flag:
 cargo generate --git https://github.com/githubusername/mytemplate.git --name myproject
 ```
 
+## Templates
+
+Templates are git repositories that contain placeholders. The current
+supported placeholders are:
+
+- `{{authors}}`: this will be filled in by a function borrowed from Cargo's source code, that determines your information from Cargo's configuration.
+- `{{project-name}}`: this is supplied by either passing the `--name` flag to the command or working with the interactive CLI to supply a name.
```

Comment:
there's also `{{crate_name}}` which is just the snake_version version of the name for convenience